java/test/jmri/util/JUnitUtil.java

```diff
@@ -95,7 +95,7 @@ public class JUnitUtil {
     /**
      * Default standard time step (in mSec) when looping in a waitFor operation.
      */
-    static final private int DEFAULT_WAITFOR_DELAY_STEP = 5;
+    static final protected int DEFAULT_WAITFOR_DELAY_STEP = 5;
```

Comment:
Why are `DEFAULT_WAITFOR_DELAY_STEP` and `DEFAULT_WAITFOR_MAX_DELAY` changed from private to protected? They doesn't seem to be used outside of JUnitUtil.

Reply:
Fixes Java doc  warnings